### ISSUE TITLE
nginx ingress controller

### ISSUE THREAD

**Issue by raphaelIl** [Raphael9292/study#3]
## TL;DR
- nginx ingress controller는 크게보면 2개 distro가 있다.
- https://www.nginx.com/products/nginx-ingress-controller/compare-models/

## 동작 방식(아키텍처)
- https://docs.nginx.com/nginx-ingress-controller/intro/how-nginx-ingress-controller-works/
- https://kubernetes.github.io/ingress-nginx/how-it-works/

## Installation
- https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/
- https://kubernetes.github.io/ingress-nginx/deploy/#aws

## Configuration
- https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/
- https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/
